src/app/components/search/search-results/search-results.component.spec.ts

```diff
@@ -164,6 +164,11 @@ describe('SearchComponent', () => {
     expect(component.formatSearchQuery(query)).toBe(query);
   });
 
+  it('should  use original user input if text contains url', () => {
+    const query = `https://ti.to/ng-conf/ng-conf2021`;
```

Comment:
what is the link is `http` ?

Reply:
If the document contains a  link  with http , we can still search it.